data/data/manifests/openshift/cloud-creds-secret.yaml.template

```diff
@@ -37,8 +37,10 @@ data:
   clouds.yaml: {{.CloudCreds.OpenStack.Base64encodeCloudCreds}}
   clouds.conf: {{.CloudCreds.OpenStack.Base64encodeCloudCredsINI}}
 {{- else if .CloudCreds.VSphere}}
-  {{.CloudCreds.VSphere.VCenter}}.username: {{.CloudCreds.VSphere.Base64encodeUsername}}
-  {{.CloudCreds.VSphere.VCenter}}.password: {{.CloudCreds.VSphere.Base64encodePassword}}
+{{- range .CloudCreds.VSphere}}
+  {{.VCenter}}.username: {{.Base64encodeUsername}}
```

Comment:
The server name is not being included here... is it needed?

Reply:
.VCenter resolves to the server name.  the goal here was to produce the credentials secret in such a way that would allow multiple vCenters to be defined.